packages/client/lib/Chain.js

```diff
@@ -197,8 +197,17 @@ export default class Chain {
    * @param {!string} from - The address from which the message is signed.
    * @return {Promise<string>} Resolves with a signed transaction.
    */
-  async sendTransaction (to, value, data, from) {
-    return this.client.getMethod('sendTransaction')(to, value, data, from)
+  async sendTransaction (to, value, data, from, extraOutputs) {
```

Comment:
`extraOutputs` parameter should not be required anymore

Reply:
good catch, removed 👍